src/components/Header/Header.tsx

```diff
@@ -1,39 +1,206 @@
-import { NavbarLarge } from './NavbarLarge';
-import { v4 as uid } from 'uuid';
-import { INavItem } from '../../models/app';
-import { NavbarSmall } from './NavbarSmall';
-import { useMediaQuery } from '@chakra-ui/react';
+import { NavbarLarge } from "./NavbarLarge";
+import { v4 as uid } from "uuid";
+import { INavItem } from "../../models/app";
+import { NavbarSmall } from "./NavbarSmall";
+import { useMediaQuery } from "@chakra-ui/react";
 
 const navItems: INavItem[] = [
   {
     id: uid(),
-    name: 'Products',
-    link: '/products',
+    name: "Products",
+    link: "/products",
+    content: [
+      {
+        id: uid(),
+        name: "Consumer",
+        link: "#",
```

Comment:
Why are the links #

Reply:
All pages have been created already. If you are not sure where to find a page, you can check the footer of the website to see different pages.